packages/flutter/test/cupertino/button_test.dart

```diff
@@ -612,32 +680,57 @@ void main() {
     expect(focusNode.hasFocus, isFalse);
   });
 
-  testWidgets('IconThemeData is not replaced by CupertinoButton', (WidgetTester tester) async {
-    const IconThemeData givenIconTheme = IconThemeData(size: 12.0);
+  testWidgets('IconThemeData falls back to default value when the TextStyle has a null size', (WidgetTester tester) async {
+    const IconThemeData defaultIconTheme = IconThemeData(size: kCupertinoButtonDefaultIconSize);
 
     IconThemeData? actualIconTheme;
 
+    // large size
```

Comment:
Nit: Comments should start with a capital letter and end with a period, as per the style guide.